checks.d/mongo.py

```diff
@@ -407,25 +448,28 @@ def _build_metric_list_to_collect(self, additional_metrics):
 
         # Defaut metrics
         metrics_to_collect.update(self.BASE_METRICS)
+        for default_metrics in self.DEFAULT_METRICS.itervalues():
+            metrics_to_collect.update(default_metrics)
 
         # Additional metrics metrics
         for option in additional_metrics:
             additional_metrics = self.AVAILABLE_METRICS.get(option)
-
             if not additional_metrics:
-                self.log.warning(
-                    u"Failed to extend the list of metrics to collect:"
-                    " unrecognized {option} option".format(
-                        option=option
+                if option in self.DEFAULT_METRICS:
+                    self.log.warning(
+                        u"`%s` option is deprecated and will be removed in a future version."
```

Comment:
Not sure if `will be removed in a future version`, as it just does nothing anymore.
You can just say that it's not needed anymore.